test/browser/home-page.test.jsx

```diff
@@ -0,0 +1,37 @@
+var should = require('should');
+var sinon = window.sinon;
+var React = require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var stubContext = require('./stub-context.jsx');
+var HomePage = require('../../pages/home.jsx');
+
+describe("HomePage", function() {
+  var blogSection;
+
+  beforeEach(function() {
+    blogSection = stubContext.render(HomePage.BlogSection);
+  });
+
+  describe("blogSection", function() {
+    it("should display featured post", function(done) {
+      process.nextTick(function() {
+        blogSection.getDOMNode().querySelector(".featured-post .entry-title").textContent.should.eql("What’s next for Thimble?");
+        done();
+      });
```

Comment:
Hmm, so for now I think this is ok, so long as you add a comment above line 18 that explains how the `process.nextTick` is used so the stub blog post loader loads the posts by then.

However, if you *do* want to go the extra mile/kilometer, that's totally awesome! It'd likely involve the following steps:

1. Writing a fake `loadBlogPosts` function that you pass as a prop to `HomePage.BlogSection`. You could use a [sinon spy](http://sinonjs.org/docs/#spies) for this, or you could actually just hand-write a fake function that looks something like this:

  ```javascript
  var respondWithBlogPosts;
  var fakeLoadBlogPosts = function(cb) {
    /* Hold on to the callback so we can trigger it later from our tests. */
    respondWithBlogPosts = cb;
  };
  ```

2. You might want to change [`stub-blog-feed-loader.js`](https://github.com/mozilla/teach.webmaker.org/blob/develop/test/browser/stub-blog-feed-loader.js) so it also exports `FAKE_POSTS`. You'll likely want your fake callback to pass this back to the component.

That's the basics... Writing tests is mostly just about "hacking" the inputs/dependencies of your code to make sure they provide the expected output, and preferably doing so in a way that's easy to understand and maintain. Feel free to experiment with other ideas if you don't like the ones I provided.